client/src/app/shared/misc/list-overflow.component.ts

```diff
@@ -0,0 +1,114 @@
+import {
+  Component,
+  Input,
+  TemplateRef,
+  ViewChildren,
+  ViewChild,
+  QueryList,
+  ChangeDetectionStrategy,
+  ElementRef,
+  ChangeDetectorRef
+} from '@angular/core'
+import { ɵBrowserDomAdapter } from '@angular/platform-browser'
+import { NgbModal, NgbDropdown } from '@ng-bootstrap/ng-bootstrap'
+import { uniqueId, lowerFirst } from 'lodash-es'
+import { ScreenService } from './screen.service'
+import { take } from 'rxjs/operators'
+
+export interface ListOverflowItemComponent {
+  label: string
+  routerLink: any
```

Comment:
I think Angular provides a type for the routerLink value  no?

Reply:
No, `routerLink` is not exposed as a type. Definition points to `string | any[]`, which I could use.